proposals/drafts/campaign_finance_filings.rst

```diff
@@ -0,0 +1,298 @@
+        Description of what this source was used for.
+
+filing_relevant_election_date
+    Date of (nearest? next?) relevant election.
+
+filing_person
+    **optional**
+    Person responsible for the filing.
+
+Committee
+---------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-committee/{{uuid}}``
+
+name
+    Name of the Committee
+
+candidate orientations
+    **optional**
+    **repeated**
+    What posture the Committee takes towards specific Candidates. Committee type
+    can be imputed based on the number and nature of these orientations.
+
+officers
+    List of Persons who are the committee officers (maybe needs indication of
+    their ranks?)
+
+status
+    Current status of the Committee.
+
+purpose
+    **optional**
+    Purpose of the Committee if any is given.
+
+Candidate Orientation
+---------------------
+
+A Committee may have no relation to any specific Candidate, but if they do have
+such a relationship, the options are complex. Hence this type.
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-candidateorientation/{{uuid}}``
+
+candidate
+    Candidate
+
+orientation
+    Enumerated among "supports", "opposes", "primary vehicle for", "surplus
+    account for" and other relationship types.
+
+Person
+------
+
+This system assumes that each Person will be generated from a specific line item
+in a Filing. As such, we may know nothing about the Person but their name. Also,
+sometimes and as far as I can see inevitably, some Persons (many in fact) will
+be corporations or other distinctly non-human entities, Supremes Court
+notwithstanding.
+
+This type is an OCD Popolo Person.
+
+Contribution (Section)
+----------------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-contribution/{{uuid}}``
+
+is_loan
+    Whether the contribution is a loan. (This type of contribution could
+    potentially merit its own Section.)
+
+is_inkind
+    Whether the contribution is in-kind. (This type of contribution could
+    potentially merit its own Section.)
+
+contribution_amount
+    Amount in Decimal of contribution.
+
+donor
+    Person making contribution.
+
+date
+    Date reported for contribution.
+
+description
+    String (may simply need repeated "notes" fields for items of this type).
+
+memo
+    String (may simply need repeated "notes" fields for items of this type).
+
+Expenditure (Section)
+---------------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-expenditure/{{uuid}}``
+
+is_transfer
+    Whether this expenditure is a transfer to another committee. (This type of
+    expenditure could potentially merit its own Section.)
+
+amount
+    Amount in Decimal of expenditure.
+
+vendor
+    Person receiving expenditure.
+
+date
+    Date reported for expenditure.
+
+description
+    String (may simply need repeated "notes" fields for items of this type).
+
+memo
+    String (may simply need repeated "notes" fields for items of this type).
+
+Amendment (Section)
```

Comment:
"Section" may not be the right abstraction for an amendment. I like @gordonje's notion of versions, though I don't know if we need version counts - we primarily care about whatever the current version is. Secondarily, for a given claim we want to be able to say which version(s) it comes from.

Maybe we should think of amendments as a linked list, allowing you to get back to previous versions and superseding versions of the same filing from wherever you are.

Should amendments (in our system) contain all the previous data from prior versions of a filing, or just a diff? I'm inclined toward the former, but could be talked out of it - seems like it's conceptually more straightforward and would be easier to use, primarily coming at the cost of (cheap and ever-cheaper) storage.

I'll add a question about how to handle amendments, and leave a stub for them in the current PR.